### ISSUE TITLE
401 Error related to extract-site-emojis

### ISSUE THREAD

**Issue by dunhampa** [viqueen/confluence-static-site#49]
I successfully used `extract` space commands to extract space pages and blogs.  I've verified this extraction via building and serving the static content later.

However, after blogs are extracted I am getting a long error message that ends with 

`data: { code: 401, message: 'Unauthorized' }`

Other parts of this error message indicate it is related to emojis:

` _currentUrl: 'https://admin.atlassian.com/gateway/api/emoji/undefined/site',
`

I believe it is related to the code on  [line 19 in this file](https://github.com/viqueen/confluence-static-site/blob/main/modules/cli/commands/extract/extract-site-emojis.ts) 

---

To confirm env settings, is there a difference between?
CONFLUENCE_API_TOKEN=
CONFLUENCE_CLOUD_TOKEN=

I am setting both values to my users Atlassian API token.  If one is from a different source it might be the culprit here.

---

I can provide more of error message but it needs to be scrubbed of some session data.

Any thoughts on how to troubleshoot?

Thank you!

**Comment by viqueen** [comment]
@dunhampa 

There indeed a difference between `CONFLUENCE_API_TOKEN` and `CONFLUENCE_CLOUD_TOKEN` , 
some of the resources like emojis and link objects are served by non-confluence specific services and those require a cookie session token in addition to the api token.

I yet have to setup a `confsite authenticate` command that will derive both tokens upon successful login, but that requires some more crafty work.

For the time being, you may look for `cloud.session.token` cookie on your browser's network tab when navigating around Confluence, and filter by `gateway/api` urls.

Also this is how I use it for my personal site https://github.com/viqueen/viqueen.org , I think there's some doco gap , we do need to run a `confsite init-site` to get the site ID ... I see that you have an undefined in that resolved url there  `.../api/emoji/undefined/site`

**Comment by dunhampa** [comment]
@viqueen 

Thanks for the  updates here.  I tried some things but still stuck.  Additional details:

I tried the init-site but received an error: 
```
you need to login first
Error: could not create api client
```

<img width="1119" alt="image" src="https://user-images.githubusercontent.com/5845872/234069303-ba547166-82a9-4f32-b12c-a07cb65ddda6.png">

To continue, I found the site ID via the browser and included it in my environment file manually in my .env file via CONFLUENCE_SITE_ID env value.

I also extracted my cloud.session.token cookie.  It looks like these values both made it to the based on the logged errors, but still getting a 401:

<img width="1790" alt="image" src="https://user-images.githubusercontent.com/5845872/234070052-f8bff603-1caf-4c5e-b357-c2917df44492.png">
